src/core/layout/scrollable-list/scrollable-list-buttons.js

```diff
@@ -39,9 +39,9 @@ const createGelButton = (scene, item) => {
 const scaleButton = args => {
     const { scene, config, gelButton } = args;
     const safeArea = scene.layout.getSafeArea();
-    const scaleFactor = (safeArea.width - config.space * 2) / gelButton.width;
+    const scaleFactor = (safeArea.width - config.space * 4) / gelButton.width;
     gelButton.setScale(scaleFactor);
     return args;
 };
 
-export { createGelButton };
+export { createGelButton, scaleButton };
```

Comment:
@FostUK are we doing exports like this or `export const`?

Reply:
it's the same thing, no? just different syntax

Reply:
That's only one export that's an object with createGelButton and scaleButton methods.

Reply:
Kinda weird that it was an object with one method before :)